sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -679,12 +679,13 @@ The `QueryResources` method also accepts a `MetricsQueryResourcesOptions`-typed
 string resourceId =
     "/subscriptions/<id>/resourceGroups/<rg-name>/providers/<source>/storageAccounts/<resource-name-1>";
 var client = new MetricsClient(
-    new Uri("https://metrics.monitor.azure.com/.default"),
+    new Uri("https://<region>.metrics.monitor.azure.com"),
     new DefaultAzureCredential());
 var options = new MetricsQueryResourcesOptions
 {
     OrderBy = "sum asc",
-    RollUpBy = { "RollUpBy=City" }
+    Size = 10,
```

Comment:
Please update the text immediately before this code sample. Note that it currently mentions `RollUpBy`.